docs/swarm/networking.md

```diff
@@ -0,0 +1,291 @@
+<!--[metadata]>
++++
+title = "Attach services to an overlay network"
+description = "Use swarm mode networking features"
+keywords = ["guide", "swarm mode", "swarm", "network"]
+[menu.main]
+identifier="networking-guide"
+parent="engine_swarm"
+weight=16
++++
+<![end-metadata]-->
+
+# Attach services to an overlay network
+
+Docker Engine swarm mode provides networking features to help you attach services to a user-defined network:
+
+* Swarm mode natively supports **overlay networks**, so you can enable
+container-to-conatiner networks. When you use swarm mode, you don't need an
+external key-value store.
+* By default, **service discovery** assigns a virtual IP address and DNS
+entry to each service in the swarm, making it available by its service name to
+containers on the same network.
+* You can configure the service to use DNS round-robin instead of a VIP.
+
+In order to use overlay networks in the swarm, you need to have the following
+ports open before you create the overlay network:
+
+* Port `7946` TCP/UDP for container network discovery.
+* Port `4789` UDP for the container overlay network.
+
+## Create an overlay network in a swarm
+
+When you run Docker Engine in swarm mode, you can run `docker network create`
+from a manager node to create an overlay network. For instance, to create a
+network named `my-network`:
+
+```
+$ docker network create \
+  --driver overlay \
+  --subnet=10.0.9.0/24 \
+  --opt encrypted \
```

Comment:
nit: be consequent in using `=` or not (perhaps using `=` is more understandable here, so that's clear that they "belong" together, but either way should be fine